app/views/vacancies/_vacancy.html.haml

```diff
@@ -5,7 +5,7 @@
     %dt= t('jobs.salary')
     %dd.double
       = vacancy.salary_range
-      = t('jobs.per_year')
+      = vacancy.working_pattern == 'Full time' ? t('jobs.per_year') : t('jobs.per_year_pro_rata')
```

Comment:
For view logic like this I'd try and group it in the presenter for reuse, testability and smaller views.